skxray/speckle_visibility/speckle_visibility.py

```diff
@@ -0,0 +1,290 @@
+# ######################################################################
```

Comment:
Can you rename move this file to the top level and rename it to not name-chcek a specific technique?  I have not seen anything domain specific in here